server/controllers/userController.js

```diff
@@ -0,0 +1,58 @@
+import model from '../models/index';
+
+const Users = model.Users;
+const Workouts = model.Workouts;
+const WorkoutType = model.WorkoutType;
+
+
+const UserController = {
+  getAllUsers(req, res) {
+    let dbQuery;
+    let result;
+    if (req.query.name) {
+      const { name } = req.query;
+      dbQuery = {
+        where: {
+          name: {
+            $iLike: `%${name}%`
+          }
+        }
+      };
+      // result = Users.findAll({
+      //   where: {
+      //     name: {
+      //       $iLike: `%${name}%`
+      //     }
+      //   }
+      // });
+    }
+    else {
+      dbQuery = {}
+      // result = Users.findAll();
+    };
+
+    return Users.findAll(dbQuery).then(users => {
+        if(!users.length) {
+          return res.status(200).send({
+            message: 'There are no users yet'
+          });
+        }
+        return res.status(200).send({
+          users
+        })
+      })
+      .catch(err => {
+         res.status(500).send({
+          message: 'Server error',
+          err
+        });
+      });
+  },
+
+  getUser(req, res) {
+    const user = req.user;
+    res.status(200).send({ user });
```

Comment:
just `res.status(200).send({ user: req.user })` should do, nor need to assign on line 53